From 036307033d5c187b3123dae46477feacbd06d0ab Mon Sep 17 00:00:00 2001
From: Joseph Macaranas <Joseph.Macaranas@amd.com>
Date: Sun, 22 Sep 2024 23:03:48 -0400
Subject: [PATCH] Allow custom package name for CI builds of torchvision

---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 4b0525d8e4..2c51ce04f2 100644
--- a/setup.py
+++ b/setup.py
@@ -42,7 +42,7 @@ CSRS_DIR = ROOT_DIR / "torchvision/csrc"
 IS_ROCM = (torch.version.hip is not None) and (ROCM_HOME is not None)
 BUILD_CUDA_SOURCES = (torch.cuda.is_available() and ((CUDA_HOME is not None) or IS_ROCM)) or FORCE_CUDA

-PACKAGE_NAME = "torchvision"
+PACKAGE_NAME = os.getenv("TORCHVISION_PACKAGE_NAME", "torchvision")

 print("Torchvision build configuration:")
 print(f"{FORCE_CUDA = }")
@@ -98,7 +98,7 @@ def get_requirements():
         except DistributionNotFound:
             return None

-    pytorch_dep = "torch"
+    pytorch_dep = os.getenv("TORCH_PACKAGE_NAME", "torch")
     if os.getenv("PYTORCH_VERSION"):
         pytorch_dep += "==" + os.getenv("PYTORCH_VERSION")

--
2.44.0.windows.1

